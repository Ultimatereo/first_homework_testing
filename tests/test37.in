5
2 4 1 3 2 1
2 0 3 0 0 0
1 5 5 0 4 4
1 5 4 1 0 0
3 1 4 5 4 0
