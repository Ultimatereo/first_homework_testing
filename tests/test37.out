0.362319
0.275362
-0.241546
-0.772947
0.86715
